kale/queue_info.py

```diff
@@ -150,6 +160,23 @@ def is_queue_empty(self, queue):
             return False
         return True
 
+    def does_queue_need_work(self, queue):
+        """Checks if a queue should be worked on.
+
+        This basically checks whether the queue is empty. However,
+        if we hit a SQS rate limit, this will assume the queue needs work.
+
+        :param TaskQueue queue: a TaskQueue object.
+        :return: True if the queue needs work; False otherwise.
+        :rtype: bool
+        """
+        try:
+            return not self.is_queue_empty(queue)
+        except boto_exception.SQSError as e:
+            if getattr(e, 'code', '') == 'RequestThrottled':
```

Comment:
Why `getattr(e, 'code', '')` instead of `e['code']`?

Reply:
Yes this could be e.code